packaging/hudi-integ-test-bundle/pom.xml

```diff
@@ -615,25 +615,25 @@
     <dependency>
       <groupId>io.confluent</groupId>
       <artifactId>kafka-avro-serializer</artifactId>
-      <version>3.0.0</version>
+      <version>5.3.4</version>
     </dependency>
 
     <dependency>
       <groupId>io.confluent</groupId>
       <artifactId>common-config</artifactId>
-      <version>3.0.0</version>
+      <version>5.3.4</version>
```

Comment:
Pull this version into to a property in the root Pom? It will also allow everyone to override it and build against different versions if needed

Reply:
done